### PR TITLE
[14.0] doc: fix typo about calendar filter

### PR DESCRIPTION
**Pull request by hparfr**
Description of the issue/feature this PR addresses:
Fix doc
Current behavior before PR:

Desired behavior after PR is merged:

@Feyensv 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by mourad-ehm — APPROVED** [review]

File: doc/reference/views.rst [hunk 1]
```diff
@@ -447,9 +447,9 @@ calendar view are:
 
     you can add a filter and save the result in the defined model, the
     filter is added in the sidebar
-  * ``filter`` and ``color``
+  * ``filters`` and ``color``
 
-    use "True" to add this field in filter in the sidebar. You can specify
+    use "True" to add this field in filters in the sidebar. You can specify
```
Comment (yelizariev) [inline doc/reference/views.rst:452]:
```suggestion
    use "True" to add this field in ``filters11 in the sidebar. You can specify
```

**Review by yelizariev — COMMENTED** [review]
Indeed, it should be ``filters`` not ``filter``. Example:

https://github.com/odoo/odoo/blob/1476c0f78281fb90e76db5eac193466b86e75a25/addons/project/views/project_views.xml#L946

**Comment by yelizariev** [comment]
Don't forget to make ``git rebase -i`` and keep squash two commits into one

**Comment by hparfr** [comment]
Oh sad,  we still have to to do these things manually. I will fix it.

**Review by yelizariev — APPROVED** [review]

**Comment by lathama** [comment]
@hparfr knowing that 14 is no longer updated, is this PR still needed?

**Comment by Feyensv** [comment]
As the documentation has been moved to a separate repository (https://github.com/odoo/documentation), and 14.0 is indeed deprecated, please open the PR on the right repo if it's still an issue (and the right people will be assigned automatically as reviewers now that we have codeowners).

**Comment by hparfr** [comment]
It looks the documentation has been updated 1,5 year after this PR : https://github.com/odoo/documentation/commit/980edde1e50873c7e8994cb9f8c95922d3e9b528 for v15

I hope with new organisation about the documentation this kind of easy pr will be merged sooner.

**Pull request closed without merging**